pkg/pullsync/pullsync.go

```diff
@@ -170,6 +170,7 @@ func (s *Syncer) Sync(ctx context.Context, peer swarm.Address, bin uint8, start
 
 		addr := offer.Chunks[i].Address
 		batchID := offer.Chunks[i].BatchID
+		batchHash := offer.Chunks[i].BatchHash
```

Comment:
Isn't it more accurately a `StampHash` than a `BatchHash`?   The latter fairly accurately describes the `BatchID` from where I read things.   Trying to keep the distinction between `Batch` and `Stamp`.

This, of course, will apply to everywhere that this appears.

Reply:
fixed